src/main/java/uk/nhs/digital/nhsconnect/nhais/mapper/FromFhirToEdifactMapper.java

```diff
@@ -0,0 +1,15 @@
+package uk.nhs.digital.nhsconnect.nhais.mapper;
+
+import org.hl7.fhir.r4.model.Parameters;
+import org.hl7.fhir.r4.model.Patient;
+import uk.nhs.digital.nhsconnect.nhais.model.edifact.Segment;
+import uk.nhs.digital.nhsconnect.nhais.parse.FhirParser;
+
+public interface FromFhirToEdifactMapper<T extends Segment> {
+    T map(Parameters parameters);
+
+    default Patient getPatient(Parameters parameters) {
+        FhirParser fhirParser = new FhirParser();
```

Comment:
Shoudln't this be injected?

Reply:
Can we fix-forward when we implement the transaction types? I think @grzegorz-robak  has introduced a new way of accessing parameters in the his PR